//package com.sincetimes.redwar.game.comnunicate.protocol;


message MsgNpc
{
    repeated string npc = 1;
    optional int32 xp = 2;
    optional int32 HQLevel = 3;
    optional string sku = 4;
}

message RseObtainNpcList
{
	repeated MsgNpc npcList = 1;
}

